tox.ini

```diff
@@ -33,12 +56,3 @@ deps = sphinx
 commands = make -C docs/docsite htmldocs
 usedevelop = True
 
-[testenv:metadata-validation]
```

Comment:
The validation env must remain separate. You may optionally specify an env dependency such as: `depends = build-dists`